extensions/wikia/UserLogin/EmailConfirmationController.class.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+class EmailConfirmationController extends WikiaController {
+	/**
+	 * Attempt to confirm the user's email address and show success or failure
+	 */
+	public function postEmailConfirmation() {
```

Comment:
I'm assuming that this method is being POSTed to since it changes the state of the server?  There should be a check here that requires the action to be POST.